### DISCUSSION TITLE
Extra div for modal buttons

### DISCUSSION THREAD

**Discussion by jeroen-hoekstra** [mantinedev/mantine#1374] [Feature requests]
Hi there,

currently we're using the confirm modal in our project. The content inside the body is dynamic and can be large. What then happens is that the buttons become invisible because they live inside the body (see screenshot). In our code we've added some custom css to always show the buttons at the bottom of the modal, but it feels hacky. Is it not possible to add an extra div (modal-footer) outside the body where these buttons can live? There is already a div for the header.

![Screenshot from 2022-05-02 15-18-59](https://user-images.githubusercontent.com/42991987/166443173-476717ca-9c86-4bf5-9033-c0c7d82e21e8.png)

**Comment by zhulien-ivanov** [comment]
The buttons are not part of the content you provide. They're in a separate `Group`: https://github.com/mantinedev/mantine/blob/a4edab9bdafbbd064516daa618c0aae0cd5a2e22/src/mantine-modals/src/ConfirmModal.tsx#L48-L58. Adding the button section as a footer is not a bad idea, though. Currently, you can solve your problem by putting your `children` inside a `ScrollArea` with fixed `height`.

**Reply by jeroen-hoekstra** [reply]
True about the Group, but the Group lives inside the body which also contains the content, so when the content grows it pushes the Group outside the visible area as shown in the screenshot. 

Thanks for the tip about the ScrollArea. Will look into that too.
